apps/errant-tower-client/src/common/styles/main.scss

```
@@ -0,0 +1,45 @@
+
+::selection {
+    background-color: $color-primary-600;
+    color: $color-dark-300;
+}
+
+:root {
+    font-family: $font-sans;
+    background-color: $color-dark-300;
+    color: $color-light-100;
+
+    font-synthesis: none;
```

Stylelint (17.7.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)